x-pack/plugins/observability/public/hooks/use_time_zone.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useUiSetting } from '@kbn/kibana-react-plugin/public';
+import moment from 'moment-timezone';
+
+export const useTimeZone = () => {
```

Comment:
I would consider moving this hook to `@kbn/kibana-react-plugin` or some other shared location